### PR TITLE
Update dependencies for common3

### PR DESCRIPTION
**Pull request by mjcarroll**
Signed-off-by: Michael Carroll <michael@openrobotics.org>

### CODE REVIEW

**Review by JShep1 — COMMENTED** [review]
I'm getting a `zmq` issue when trying to build `ign_transport` (https://github.com/ignitionrobotics/ign-transport/pull/196#pullrequestreview-529338883).  Not sure if the issue I'm seeing is here or in `ign_transport`

File: repositories.bzl [hunk 1]
```diff
@@ -55,28 +163,12 @@ def ign_math_repositories():
...
-        path = "/usr/lib/x86_64-linux-gnu/glib-2.0/include",
-        build_file_content = """
-package(default_visibility = ["//visibility:public"])
-cc_library(
-    name = "headers",
-    hdrs = glob(["**/*.h"])
-)
-""",
-    )
+    dl()
```
Comment (JShep1) [inline repositories.bzl:166]: Think this needs to be deleted; I noticed you deleted the corresponding function

**Review by JShep1 — CHANGES_REQUESTED** [review]

File: repositories.bzl (hunk 1 above)
Comment (mahiuchun) [reply repositories.bzl:166]: +1

Comment (mjcarroll) [reply repositories.bzl]: Removed in latest.

**Comment by mjcarroll** [comment]
> I'm getting a zmq issue when trying to build ign_transport (ignitionrobotics/ign-transport#196 (review)). Not sure if the issue I'm seeing is here or in ign_transport

I believe that I picked this up from cppzmq, I removed it because I don't believe that it is actually used.

**Review by JShep1 — APPROVED** [review]

**Comment by JShep1** [comment]
Good for me now

File: BUILD.bazel [hunk 2]
```diff
@@ -23,7 +23,8 @@ py_binary(
 
 cc_library(
     name = "utilities",
-    srcs = [
+    hdrs = [
+        "ignition/utilities/ExtraTestMacros.hh",
```
Comment (mahiuchun) [inline BUILD.bazel:27]: This should be `ignition/utilities/detail/ExtraTestMacros.hh`?

**Pull request merged**